package.json

```diff
@@ -35,11 +36,20 @@
     "chai-as-promised": "^7.0.0",
     "chai-bignumber": "^2.0.0",
     "coveralls": "^2.13.1",
-    "ethereumjs-util": "^5.1.2",
+    "eslint": "^4.11.0",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-plugin-import": "^2.8.0",
+    "eslint-plugin-node": "^5.2.1",
+    "eslint-plugin-promise": "^3.6.0",
+    "eslint-plugin-standard": "^3.0.1",
     "ethereumjs-testrpc": "^6.0.1",
+    "ethereumjs-util": "^5.1.2",
     "mocha-lcov-reporter": "^1.3.0",
     "solidity-coverage": "^0.2.2",
     "truffle": "^4.0.0",
     "truffle-hdwallet-provider": "0.0.3"
+  },
+  "dependencies": {
+    "dotenv": "^4.0.0"
```

Comment:
Is this needed in `dependencies` or could it go in `devDependencies`? It only seems necessary if you want to run the tests.

Reply:
It's required at runtime for `truffle.js`, so theoretically it belongs in dependencies. But we really only use truffle for tests, since anyone using this project is including it as a dependency and using their own truffle.

It's up to you. We can also remove it, but being able to include a `.env` file for things like `MNEMONIC` and infura api keys is a lot easier than making sure to export them in your environment or include them in the command line when you run `truffle`

Reply:
Let's leave it this way. `dotenv` has no production dependencies anyway.